runtime/src/bank.rs

```diff
@@ -359,6 +361,54 @@ impl Bank {
         Self::new_with_paths(&genesis_config, Vec::new(), &[])
     }
 
+    pub fn new_from_1_0(bank_1_0: Bank1_0) -> Self {
```

Comment:
Can you put this in `Bank1_0` instead?  Make Bank1_0 be able to convert into Bank.  This way when we don't care about Bank1_0, we delete bank_1_0.rs and be done with it

Reply:
ref: 3bc9789e8dd8852556ccf098f455b487b534b479

Reply:
yup thats fine, I'll just have to move `bank_1_0.rs` to be a submodule of `bank` so that it has access to set all its private fields

Reply:
Actually maybe never mind.  Sorry I didn't spot that this PR was on v1.0, so there's no ongoing support burden once we hit v1.2 so meh.

Reply:
hehe oops, beat you to it :D